src/base/static/state/ducks/place-config.js

```diff
@@ -26,11 +26,32 @@ export const placeConfigPropType = PropTypes.shape({
 });
 
 // Actions:
-const SET_CONFIG = "place-config/SET";
+const LOAD = "place-config/LOAD";
 
 // Action creators:
-export function setPlaceConfig(config) {
-  return { type: SET_CONFIG, payload: config };
+export function setPlaceConfig(config, user) {
```

Comment:
Thanks for using the LOAD terminology. I would also suggest renaming this action creator to `loadPlaceConfig`